package.json

```diff
@@ -41,6 +41,7 @@
     "eslint-plugin-promise": "^4.2.1",
     "eslint-plugin-standard": "^4.0.0",
     "eslint-plugin-vue": "^7.20.0",
+    "husky": "^4.3.8",
```

Comment:
Husky seems not used in this branch.

Maybe you should set up (with git hooks ?) on another branch.

Reply:
@0live Du coup il faut l'enlever du package.json également, non ?

Reply:
Ah oui désolé, pour la coquille...